base/stats.go

```diff
@@ -30,6 +30,7 @@ const (
 
 	NamespaceKey                 = "sgw"
 	ResourceUtilizationSubsystem = "resource_utilization"
+	ErrorsSubsystem              = "errors"
```

Comment:
I'm wondering if 'errors' is just a bit too generic.  What do you think about 'config' instead?  I think there are candidates for new stats in the db registry functionality that would also make sense in a category like that, and also in the config polling/bootstrap code.